doc/api/cli.md

```diff
@@ -546,6 +546,14 @@ added: v6.0.0
 
 Silence all process warnings (including deprecations).
 
+### `--node-memory-debug`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enable extra debug checks for memory leaks in Node.js itself. This is usually
+not meaningful when working on Node.js applications.
```

Comment:
I wonder if it's easier to understand if we say "This is usually only useful for developers debugging Node.js itself"

Reply:
I also wonder if "Node.js itself" here and below might be better as "Node.js internals" or something like that? Non-blocking, though, as I'm not actually sure that's better...

Reply:
Sure, done :+1: